src/backend-query.js

```diff
@@ -2,18 +2,18 @@
 import hetmechDefinitions from './definitions.json';
 // url for metagraph (node and edge metatypes and abbreviations)
 const metagraphUrl =
-  'https://raw.githubusercontent.com/hetio/hetionet/6e08d3039abaad8f6dafe26fe3b143773b0d7e51/hetnet/json/hetionet-v1.0-metagraph.json';
+    'https://raw.githubusercontent.com/hetio/hetionet/6e08d3039abaad8f6dafe26fe3b143773b0d7e51/hetnet/json/hetionet-v1.0-metagraph.json';
```

Comment:
Why the indentation is 4 now? (Same issue on lines 8, 11, 13, 16).

Reply:
Fixed it. `backend-query.js` accidentally got changed when I was playing around with your fuzzy search. The file shouldn't be in the diff at all.